eq-author/src/App/page/Design/answers/MultipleChoiceAnswer/MultipleChoiceAnswer.test.js

```diff
@@ -70,6 +74,7 @@ describe("MultipleChoiceAnswer", () => {
   beforeEach(() => {
     store = createMockStore();
     mockHandlers = {
+      ...mockHandlers,
```

Comment:
Can we just add onMoveOption here rather than adding it to mockhandlers then spreading it out down here?

Reply:
changed